main.go

```diff
@@ -0,0 +1,12 @@
+package main
```

Comment:
Go won't let you have two files in the same directory with different package names.

the way many big projects work around this is that they create a directory
cmd/pets/main.go

which contains the main package for the pets command

https://github.com/golang-standards/project-layout

Reply:
Thank you!